src/middleware/companyAuth.middleware.ts

```diff
@@ -6,15 +6,28 @@ import { NextFunction, Request, RequestHandler, Response } from 'express'
 import AuthConfig from 'app/models/authConfig'
 import Optional from 'app/models/optional'
 import DissolutionSession from 'app/models/session/dissolutionSession.model'
+import { HEALTHCHECK_URI, ROOT_URI, SEARCH_COMPANY_URI, VIEW_COMPANY_INFORMATION_URI, WHO_TO_TELL_URI } from 'app/paths'
 import JwtEncryptionService from 'app/services/encryption/jwtEncryption.service'
 import SessionService from 'app/services/session/session.service'
 
 const OATH_SCOPE_PREFIX = 'https://api.companieshouse.gov.uk/company/'
 
+const COMPANY_AUTH_WHITELISTED_URLS: string[] = [
+  `${ROOT_URI}/`,
```

Comment:
For the root URI (landing page), req.url is "/close-a-company/". So thought this was cleaner than messing with any config or changing the consts in paths.ts